src/test/java/com/epam/petclinic/clinic/controller/ClinicControllerTest.java

```diff
@@ -57,7 +57,7 @@
 
     @Test
     public void testGetAllClinics() {
-        List<Clinic> clinics = restTemplate.getForObject(getHost() + CLINIC_PATH, List.class);
+        List clinics = restTemplate.getForObject(getHost() + CLINIC_PATH, List.class);
```

Comment:
It is highlighted in idea and  build log has
Note: D:\nasdaq\spring-cloud\petclinic-clinic\src\test\java\com\epam\petclinic\clinic\controller\ClinicControllerTest.java uses unchecked or unsafe operations.